### PR TITLE
Nanopaste Fixes #2892

### LINKED ISSUE #2892: Nanopaste does nothing when trying to heal artificial organs.
Just doesn't work.

### PR DESCRIPTION
**Pull request by fernandos33**
Nanopaste fixed (now you can fix mechanical organs, and if you open robotic limbs you can fix them with nanopaste as if it was welder or wire. not implemented by me just fixed)
Fixes #2892
Signed-off-by: fernandos33 <ois333@yahoo.com>

### CODE REVIEW

File: code/game/objects/items/stacks/nanopaste.dm [hunk 1]
```diff
@@ -7,10 +7,15 @@
 	matter = list(MATERIAL_PLASTIC = 10)
 	origin_tech = list(TECH_MATERIAL = 4, TECH_ENGINEERING = 3)
 	amount = 10
+	w_class = ITEM_SIZE_SMALL
+	var/heal_brute = 0
+	var/heal_burn = 0
```
Comment (ACCount12) [inline code/game/objects/items/stacks/nanopaste.dm]: Why add those two variable?

File: code/game/objects/items/stacks/nanopaste.dm [hunk 2]
```diff
@@ -44,8 +49,8 @@
 				else
 					user << SPAN_NOTICE("Nothing to fix here.")
 		else
-			if (can_operate(H))
+			if (can_operate(H))        //Checks if mob is lying down on table for surgery
 				if (do_surgery(H,user,src))
 					return
 			else
-				user << SPAN_NOTICE("Nothing to fix in here.")
+				user << SPAN_NOTICE("The [S.name] is badly damaged, you'll need more than a nanopaste!")
```
Comment (ACCount12) [inline code/game/objects/items/stacks/nanopaste.dm]: Why this change?

File: code/game/objects/items/stacks/nanopaste.dm (hunk 1 above)
Comment (ACCount12) [inline code/game/objects/items/stacks/nanopaste.dm]: Why add those two variable?

File: code/modules/surgery/organs_internal.dm [hunk 3]
```diff
@@ -98,6 +98,71 @@
...
 
+/datum/surgery_step/internal/fix_organ_rob
+	allowed_tools = list(
+	/obj/item/stack/nanopaste = 100
+	)
+
+	min_duration = 70
+	max_duration = 90
+
+	can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
```
Comment (ACCount12) [inline code/modules/surgery/organs_internal.dm]: Do not use relative pathing. Procs should be `/datum/surgery_step/internal/fix_organ_rob/can_use(` and such.

**Comment by fernandos33** [comment]
i|ll see if i can add it inside of the regular organ internal one let me try (don|t wait for it, will take a lot fot time, i think)

**Comment by fernandos33** [comment]
so far i did not made enough progess as to change the whole code so theres a need for it to be separated from other surgeries (just like with the organic organ repair). and i need to check on the relative pathing, will do soon,

**Comment by zamolxius** [comment]
why does this also add smokes to vending machines?  did you mess up the commits or im wrong?

**Comment by fernandos33** [comment]
damn i|m making a mess, i|m going to bed and i|ll get back to fixing it all tomorrow the 3 prs. one by one. (at least its not in the export thing wich is the most important)

**Comment by fernandos33** [comment]
fixed the cigarette being in this commit mistake. now going to fix the others PR

**Comment by fernandos33** [comment]
> I'm not sure if adding a new copypasted surgery step is the right approach. Why not change the step that's used for healing other organs instead?

cause from what i saw if i change the tools and add nanopaste theres no way to later tell the code what tool is being used in that surgery cause the surgery steps is a °formula° from the surgery.dm
